rand_distr/src/zipf.rs

```diff
@@ -0,0 +1,322 @@
+// Copyright 2021 Developers of the Rand project.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! The Zeta and related distributions.
+
+use num_traits::Float;
+use crate::{Distribution, Standard};
+use rand::{Rng, distributions::OpenClosed01};
+use core::fmt;
+
+/// Samples floating-point numbers according to the zeta distribution.
+///
+/// The zeta distribution is a limit of the [`Zipf`] distribution.
+///
+/// # Example
+/// ```
+/// use rand::prelude::*;
+/// use rand_distr::Zeta;
+///
+/// let val: f64 = thread_rng().sample(Zeta::new(1.5).unwrap());
```

Comment:
I propose that the distribution should be changed to integers, so here `f64` would change.

Reply:
See below.

Reply:
Okay! I understand.